addons/website/static/src/scss/website.scss

```diff
@@ -1221,6 +1221,9 @@ header {
         @include o-apply-colors('header-sales_three-custom');
         @include o-add-gradient('menu-secondary-gradient');
     }
+    #wrapwrap .o_header_sales_three_small_links {
+        --nav-link-font-size: #{$font-size-sm};
```

Comment:
`$font-size-sm` looks correct when the header uses the default font-size, but as soon as it's customised proportions will mismatch. Also, customizing the header font-size seems to force inner items to `inherit`,  invalidating your rule 🤷🏻‍♂️ 

![image](https://github.com/user-attachments/assets/6aa3a3b5-1ba0-4220-8f62-6707b1d81ab1)

Reply:
Good catch, thanks ! 
I've forced the font-size to be always proportional to header font size, what do you think ?

As we discussed, I also rebased on `master`.